ec/src/models/mod.rs

```diff
@@ -68,6 +65,15 @@ pub trait TEModelParameters:
         copy *= &Self::COEFF_A;
         copy
     }
+
+    /// Checks that the current point is in the prime order subgroup given
+    /// the point on the curve.
+    fn is_in_correct_subgroup_assuming_on_curve(
+        item: &twisted_edwards_extended::GroupAffine<Self>,
+    ) -> bool {
+        item.mul_bits(BitIteratorBE::new(Self::ScalarField::characteristic()))
+            .is_zero()
+    }
 }
 
 /// Model parameters for a Montgomery curve.
```

Comment:
Why isn't there one for a Montgomery curve?

Reply:
Mostly because we only ever instantiate the Montgomery model of a curve in a very specific circumstance (inside a particular hash function), and so we don't need it generally).